src/AWS.Deploy.CLI/Commands/CommandFactory.cs

```diff
@@ -343,6 +347,7 @@ private Command BuildDeploymentProjectCommand()
                     OrchestratorSession session = new OrchestratorSession(projectDefinition);
 
                     var targetApplicationFullPath = new DirectoryManager().GetDirectoryInfo(projectDefinition.ProjectPath).FullName;
+                    _deploymentManifestEngine.TargetApplicationFullPath = targetApplicationFullPath;
```

Comment:
I don't like that we are setting this here. It should either be part of the constructor or passed to the function that needs it. (Following up on a later comment)

Reply:
Fixed. Passing it as parameter to methods which need this.